.eslintrc.json

```
@@ -0,0 +1,176 @@
+{
+	"ignorePatterns": [
+		"dist/"
+	],
```

I'm noticing just now that this isn't needed.
```suggestion
{
```